docs/GUIDE.md

```diff
@@ -0,0 +1,229 @@
+# A Warrior's Guide to Thunderdome
+
+Thunderdome is a fun way to perform agile scrum planning. See Wikipedia.
+
+As a new warrior in this realm, let this be your guide to Battles. First, we need to know who you are.
+
+## Register (optional)
+
+Create a new account, or join as guest.
+
+Having an account lets you save your battles and more.
+
+- Name
+This will be visible to others.
+- Email (for account)
+- Password (for account)
+Use a strong password. Type it again to confirm
+
+You will receive an email to confirm your new account.
+
+## Login
+
+Use the email/password you created when registering.
+
+- Email
+- Password
+
+_OIDC Providers coming soon._
+
+### Password Retrieval
+
+Forgot your password? Thunderdome can send a link to your email.
+
+## Profile
+
+Warrior, it is all about you! Control your Thunderdome experience.
+
+### Details
+
+- Name
+This will be visible to others.
+- Email
+Update your account email, or enter one if you are a guest.
+- Country (optional)
+- Locale, default: English
+8 locales to choose from.
+- Company (optional)
+- Job Title (optional)
+- Theme, default: auto
+If you prefer a darker or lighter interface, you may choose it here.
```

Comment:
Auto lets the operating system and browser define dark or light, if supported.

Reply:
```suggestion
The default lets the operating system and browser define dark or light, if supported. If you prefer a darker or lighter interface, you may choose it here.
```